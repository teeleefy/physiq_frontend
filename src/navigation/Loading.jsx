import {  Card, CardBody, CardTitle, CardText } from "reactstrap";
import './styles/Loading.css'

function Loading(){
    return(
        <Card className="Loading">
            <CardBody>
                <CardTitle className="Loading-title">Loading ...</CardTitle>
            </CardBody>
        </Card>
    )
}

export default Loading;