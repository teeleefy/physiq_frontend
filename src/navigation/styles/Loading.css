.Loading {
    background-color: rgb(243, 165, 22);
    border-color: rgb(243, 165, 22);
    color: white;
    width: 30vw;
    text-align: center;
}

.Loading-title {
    font-size: 2rem;
    font-weight: 600;
}