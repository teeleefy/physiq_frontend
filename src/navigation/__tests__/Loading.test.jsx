import { renderWithRouter, screen } from '../../../__tests__/test-tools';
import Loading from '../Loading'

describe('Loading', () => {
  it('renders the Loading component', () => {
    renderWithRouter(<Loading />)
    screen.debug(); // prints out the jsx in the Loading component unto the command line
  })
})

it("matches snapshot", function() {
    const { asFragment } = renderWithRouter(<Loading />);
    expect(asFragment()).toMatchSnapshot();
  });

