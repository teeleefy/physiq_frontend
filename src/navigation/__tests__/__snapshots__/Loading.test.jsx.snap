// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Loading card"
  >
    <div
      class="card-body"
    >
      <div
        class="Loading-title card-title"
      >
        Loading ...
      </div>
    </div>
  </div>
</DocumentFragment>
`;
